examples/sdl2/peanut_sdl.c

```diff
@@ -17,6 +17,8 @@
 #	include "minigb_apu/minigb_apu.h"
 #endif
 
+#define PEANUT_FULL_GBC_SUPPORT 1
```

Comment:
This will have to be removed before the merge, or maybe this pull request could be merged in to a working branch.